de.json

```
@@ -0,0 +1,847 @@
+        "World.Category.MyWorlds" : "Meine Welten",
+        "World.Category.Custom" : "Einstellungen",
+
+        "World.AccessLevel.Anyone" : "Jeder",
+        "World.AccessLevel.RegisteredUsers" : "Registrierte Nutzer",
+        "World.AccessLevel.Friends" : "Freunde",
+        "World.AccessLevel.LAN" : "LAN",
+        "World.AccessLevel.Private" : "Privat (auf Einladung)",
+
+        "World.SortParameter.SearchScore" : "Such Score",
+        "World.SortParameter.Name" : "Name",
+        "World.SortParameter.OpenedWorldCount" : "Geöffnete Welten",
+        "World.SortParameter.TotalSessionCount" : "Anzahl Sitzungen",
+        "World.SortParameter.TotalUserCount" : "Anzahl Nutzer",
+        "World.SortParameter.CreationDate" : "Erstellt am",
+        "World.SortParameter.LastUpdateDate" : "Zuletzt geändert am",
+        "World.SortParameter.FirstPublishTime" : "Veröffentlicht am",
+        "World.SortParameter.TotalVisits" : "Anzahl Besuche",
+
+        "World.Filter.DataSources" : "Datenquellen:",
+        "World.Filter.OpenedWorlds" : "Geöffnete Welten",
+        "World.Filter.ActiveSessions" : "Aktive Sitzungen",
+        "World.Filter.PublishedWorlds" : "Veröffentlichte Welten",
+        "World.Filter.LocalWorlds" : "Lokale Welten",
+        "World.Filter.BaseSearch" : "Suche:",
+        "World.Filter.ByOwner" : "Nach Eigentümer:",
+        "World.Filter.UserOwner" : "Angemeldeter Nutzer",
+        "World.Filter.GroupOwner" : "Eigentümer ist Gruppe",
+        "World.Filter.SubmittedToGroup" : "Gruppe:",
+        "World.Filter.Featured" : "Nur Featured",
+        "World.Filter.MyWorlds" : "Meine Welten",
+        "World.Filter.SessionFiltering" : "Sitzungs-Filter:",
+        "World.Filter.MinimumUsers" : "Minimale Anzahl Nutzer: <b>{n}</b>",
+        "World.Filter.MinimumFriends" : "Minimale Anzahl Kontakte: <b>{n}</b>",
+        "World.Filter.OnlyHeadless" : "Nur Headless Hosts",
+        "World.Filter.MaxAccessLevel" : "Maximale Zugriffsberechtigung:",
+        "World.Filter.MinimumUptime" : "Minimale Betriebszeit:",
+        "World.Filter.Sorting" : "Sortierung:",
+
+        "Tools.AvatarCreator" : "Avatar Wizard",
+        "Tools.FullBodyCalibrator" : "Ganzkörper Kalibrator",
+        "Tools.CameraStreaming" : "Kamera / Streaming",
+        "Tools.CreateNewWorld" : "Erstelle neue Welt",
+        "Tools.DepositNCR" : "Einzahlen von NCR",
+        "Tools.WithdrawNCR" : "Abheben von NCR",
+        "Tools.Debug" : "Debuggen",
+
+        "Options.FreeformDash.On" : "Freiform Dash: An",
+        "Options.FreeformDash.Off" : "Freiform Dash: Aus",
+        "Options.SeatedMode.On" : "Sitzend Modus: An",
+        "Options.SeatedMode.Off" : "Sitzend Modus: Aus",
+
+        "Indicator.OnlineUsers" : "Online Nutzer: ~{num_total} ({num_registered})",
+        "Indicator.ServerStatus" : "<color=#fff>Server Status:</color> {status, select, OK {Good} Langsam {Slow} Außer Betrieb {Down} Kein Internet {No Internet}} ({response_ms} ms)",

```

This is swapped. The terms inside of the { } need to be translated here. The words outside of them are what is being matched against with the source variable.

---

```suggestion
        "Indicator.ServerStatus" : "<color=#fff>Server Status:</color> {status, select, Good {OK} Slow {Langsam} Down {Außer Betrieb} NoInternet {Kein Internet}} ({response_ms} ms)",
```
Would be the correct entry in this case.